### PR TITLE
Provide more details to client when encountering errors on import

### PR DESCRIPTION
**Pull request by scottwittenburg**
Attempt to provide more information to the client when errors are encountered importing data on the server.  Sometimes this will be useful, as in this case when the nifti folder doesn't exist:

import_error_more_useful

However, in other cases (e.g. csv file missing columns, etc.), the errors we show the client might not be as intuitive:

import_error_less_useful

### CODE REVIEW

**Comment by scottwittenburg** [comment]
@dchiquito This additionally addresses the issue you discovered and attempted to fix with #100.

If you have ideas on how to make the presentation nicer, please feel free to make suggestions or even push commits to the branch.

**Comment by aashish24** [comment]
@scottwittenburg this is great. In the second case, can it not output the fact that this is coming from CSV validation step?

**Comment by scottwittenburg** [comment]
> In the second case, can it not output the fact that this is coming from CSV validation step?

I think that defeats the whole point of this PR.  The reason we're showing stack traces to the user (not ideal) is to avoid spending development time at the current juncture trying to catch every possible problem on the server and report it properly.

**Comment by aashish24** [comment]
> > In the second case, can it not output the fact that this is coming from CSV validation step?
> 
> I think that defeats the whole point of this PR. The reason we're showing stack traces to the user (not ideal) is to avoid spending development time at the current juncture trying to catch every possible problem on the server and report it properly.

Sure, I agree with that, I was just wondering if there is a an easier way to get more details on the stack trace. Just curiosity, no action needed.

**Comment by scottwittenburg** [comment]
> I was just wondering if there is a an easier way to get more details on the stack trace

Definitely, if you see a way to do this in the PR, please let me know.  Another pair of eyes on the code is always useful and appreciated.

**Comment by aashish24** [comment]
I know that something like stackprinter exists that can help provide more context and information (https://github.com/cknd/stackprinter) but I am unsure how easy it would be include in this PR (would it be a quick work or not)

File: server/miqa_server/session.py [hunk 1]
```diff
@@ -135,125 +137,126 @@ def _getSessions(self):
         Description('')
         .errorResponse())
     def dataImport(self, params):
-        user = self.getCurrentUser()
-        importpath = os.path.expanduser(Setting().get(importpathKey))
-        if not os.path.isfile(importpath):
-            raise RestException('import path does not exist ({0}'.format(importpath), code=404)
```
Comment (dchiquito) [inline server/miqa_server/session.py:141]: I notice this check was removed. Was that intentional?

**Review by dchiquito — APPROVED** [review]

File: server/miqa_server/session.py (hunk 1 above)
Comment (scottwittenburg) [reply server/miqa_server/session.py:141]: Thanks for taking a careful look @dchiquito.  Yes, it was intentional for two reasons, but please double-check my reasoning:  First, the api method `POST miqa_setting/datapath` checks that the import path you configure is valid, so this couldn't really ever get hit anyway.  Second, the spirit of this thing is to capture any possible error that could be raised and put it in front of the user, so raising an exception here didn't fit within that goal, at least to my thinking.

Let me know what you think.

Comment (dchiquito) [reply server/miqa_server/session.py:141]: Sounds reasonable to me!

**Comment by dchiquito** [comment]
LGTM

**Pull request merged**